extension/goplugin/util.go

```diff
@@ -263,12 +260,20 @@ func sliceToMap(context map[string]interface{}, fieldName string, field reflect.
 	return nil
 }
 
+var (
+	jsonMapperInitOnce sync.Once
+	jsonMapper         *reflectx.Mapper
```

Comment:
I think 
```Go
var (
    jsonMapper = reflectx.NewMapper("json")
)
```
will do

Reply:
Done. This was too obvious for me to see ;)